### PR TITLE
Add note to have Designers review visual changes

### PR DESCRIPTION
**Pull request by chao-xian**
## What
Adds a reminder to have visual changes reviewed by Designers.

## Why
We need to get the review done earlier in our workflow.

### CODE REVIEW

**Comment by andysellick** [comment]
Can I ask why this is needed? The link to the preview deployment is always on the PR.

**Review by injms — CHANGES_REQUESTED** [review]
I don't think that this should be merged in until other frontend devs working on the components gem have had a chance to understand why this is needed - can you mention it at the next frontenders meeting?

**Comment by chao-xian** [comment]
@injms have added "if required" to the note... 

@andysellick this isn't for every case, but to help Designers who may get pointed to the PR but feel a bit lost. We've just been missing points in time to do reviews as there's no established workflow for it. It's not for every PR, but having a note here at the point of PR creation is a good opportunity for devs to be reminded.

**Comment by alex-ju** [comment]
I'm generally in favour of this note, but do we have a commitment from the design community on checking these pull requests as they are being raised? In previous teams I worked, when defining frontend cards we added a requirement for design and product to check the change before merging the change – now, I appreciate this may not be the case in every team – with some of them unfortunately not even having access to a designer.

**Comment by chao-xian** [comment]
Thanks @alex-ju - yes I'm very aware not every team won't have a designer (or not even one that's full time on the team) so it's a pragmatic approach. This is really more of a _reminder_.

**Comment by owenatgov** [comment]
An alternative proposal based on discussions off github:

```
<!-- If a change results in visual updates, include detailed screenshots that show the various states. -->
<!-- If a visual change is significant, consider running your changes past a designer -->
<!-- If your team doesn't have access to a designer or you're making a change outside a team, you can ask on the #design slack channel. -->
<!-- Not every visual change will require a designer's input -->
<!-- If in doubt, try to ask a designer -->
<!-- To help designers, include a link to the heroku preview app generated by the PR -->
```

Summary of changes:

- Soften language so it doesn't look like every bitty visual change needs a designer to drop everything and take a look and to stop FE's feeling like they're blocked
- Make allowances for devs who don't have a designer available to them so that we are trusting FE's to make good UI choices when they don't have a designer available to them at a given time
- General language-smithing

**Comment by chao-xian** [comment]
@owenatgov Nice! Thank you. I'm just a little bit wary of this nudge getting too long. Is `...if required` enough?

**Comment by owenatgov** [comment]
@chao-xian I think that's fine, less verbose 👍

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -6,4 +6,13 @@
 <!-- What are the reasons behind this change being made? -->
 
 ## Visual Changes
-<!-- If the change results in visual changes, show a before and after -->
+<!-- If change results in visual changes, include detailed screenshots that show the various states. -->
+
+<!-- Please ensure that the changes are reviewed by a Designer if required. -->
+<!-- To help Designers, please include a link to specific elements to review, -->
+<!-- for example to https://components.publishing.service.gov.uk/public -->
```
Comment (injms) [inline docs/pull_request_template.md:13]: The preview URL pattern has been updated so it's now predictable - could this be included in the example link?

```suggestion
<!-- for example to https://components-gem-pr-[PULL REQUEST NUMBER].herokuapp.com/public -->
```

**Review by alex-ju — APPROVED** [review]

**Comment by chao-xian** [comment]
Thanks everyone!

**Pull request merged**